### PR TITLE
fix compilation errors

### PR DESCRIPTION
**Pull request by themr0c**
Fix compilation errors due to headings for https://github.com/eclipse/che-docs/pull/901

### CODE REVIEW

**Review by sleshchenko — APPROVED** [review]
LGTM

**Comment by sleshchenko** [comment]
@themr0c Would you like to wait for Robert's approval here or let's merge it and continue review in the main PR https://github.com/eclipse/che-docs/pull/901. WDYT?

**Comment by themr0c** [comment]
Let's merge ;)

**Pull request merged**
